.github/workflows/build.yml

```diff
@@ -526,6 +526,31 @@ jobs:
       - run: cargo install taplo-cli --locked
       - run: taplo format --check
 
+  typos:
+    name: Typos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - run: cargo install typos-cli --locked
+      # XXX BRODY TODO REMOVE - SORRY REMOVE TYPO TEST: UPDATE
+      # XXX BRODY REMOVE ANOTHER TYPO TEST: TYPS
+      # XXX XXX typos did not catch this:   ^^^^ - ???
+      # XXX TODO REMOVE ANOTHER TYPO BELOW ==> "seem"
+      # NOTE: separate steps seem sorry seeemed to be needed to check some of these filename patterns
+      # - run: typos .*.toml
+      # - run: typos .github/workflows/*.yml
+      # - run: typos
+      # - run: admin/typoss
+      # OOPS SRRY I MEANT SORRY
+      # XXX: ^^^^ - typos doesn't seem to catch this pattern at this point
```

Comment:
I reported this uncaught typo here: https://github.com/crate-ci/typos/issues/1221#issuecomment-2648925186